nav2_map_server/test/unit/test_map_io_3d.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2020 Shivam Pandey
```

Comment:
The testing is quite good, I have no comments / requests or complaints. Good work!